gammapy/spectrum/models.py

```diff
@@ -38,10 +37,6 @@ class SpectralModel(object):
     `~gammapy.spectrum.models.PowerLaw`.
     """
 
-    def __repr__(self):
```

Comment:
Why did you remove this `__repr__`? I think it's useful to have it. I guess in the current implementation it could go to the `Model` base class as well?

Reply:
The current repr didn't show any useful information. IMO if we don't have any useful info beyond the class name that the default repr shows, we should just do nothing and show the default repr.

Yes, we probably should add `Model.__repr__` and have it show something like number of parameters or parameter names or ... it'll need some thought and discussion and once we agree on what would be useful to show we can add it back.

For that it'll help if this sub-class doesn't overwrite it and we don't notice, so I prefer to remove that now.